Xamarin.Essentials/Connectivity/Connectivity.ios.tvos.mac.reachability.cs

```diff
@@ -33,8 +33,10 @@ internal static NetworkStatus RemoteHostStatus()
                 if (!IsReachableWithoutRequiringConnection(flags))
                     return NetworkStatus.NotReachable;
 
+#if __IOS__
```

Comment:
hmm... I wonder if we should add `IsWWAN` to XM
In general we add _all_ enum values and mark them as unavailable on the platform (where they don't work)
Just so you don't have to add all those defines in _user_ code